pom.xml

```diff
@@ -218,6 +223,11 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-webmvc</artifactId>
+			<version>6.0.8</version>
```

Comment:
How is it possible that you update spring-webmvc but the DEPENCY File is not reflecting this change?

Have you not updated/run DASH again? :(

Reply:
@sachinargade123 : Please check

Reply:
Yes, DASH tool was not run for this commit,  DEPENCY file update for next PR https://github.com/eclipse-tractusx/autosetup-backend/pull/31.